fontforge/python.c

```diff
@@ -838,14 +838,9 @@ static PyObject *PyFF_UnicodeBlockNameFromLib(PyObject *UNUSED(self), PyObject *
     return( ret );
 }
 
+/* Deprecated */
 static PyObject *PyFF_UnicodeNamesListVersion(PyObject *UNUSED(self), PyObject *UNUSED(args)) {
-/* If the library is available, then return the Nameslist Version number */
-    char *temp;
-
-    if ( (temp=unicode_library_version())==NULL ) {
-	temp=malloc(1*sizeof(char)); *temp='\0';
-    }
-    PyObject *ret=Py_BuildValue("s",temp); free(temp);
+    PyObject *ret=Py_BuildValue("s","");
```

Comment:
We should probably just return `20190701` until this is actually removed (same for native scripting), we can also add something like

```
 LogError(_("UnicodeNamesListVersion is deprecated and will be removed in a future release\n"));
```

Reply:
I recommend starting to do these using python semantics, as in:
```
PyErr_Warn(PyExc_DeprecationWarning, "(message)");
```